pkgs/development/python-modules/mocket/default.nix

```diff
@@ -78,15 +79,19 @@ buildPythonPackage rec {
 
   _darwinAllowLocalNetworking = true;
 
-  disabledTests = [
-    # tests that require network access (like DNS lookups)
-    "test_truesendall_with_dump_from_recording"
-    "test_aiohttp"
-    "test_asyncio_record_replay"
-    "test_gethostbyname"
-    # httpx read failure
-    "test_no_dangling_fds"
-  ];
+  disabledTests =
+    [
+      # tests that require network access (like DNS lookups)
+      "test_truesendall_with_dump_from_recording"
+      "test_aiohttp"
+      "test_asyncio_record_replay"
+      "test_gethostbyname"
+      # httpx read failure
+      "test_no_dangling_fds"
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
Should add a note on here about why you're disabling the test, also makes it easier to spot when this should be removed ie a bump to a newer fixed version. Instead of leaving it perpetually disabled

Reply:
agree -- wasn't clear it was due to upstream bug initially. added.